### PR TITLE
Fix NPE on swap model

### PR DESCRIPTION
**Pull request by ademar111190**

### CODE REVIEW

File: lib/bloc/reverse_swap/reverse_swap_model.dart [hunk 1]
```diff
@@ -66,11 +66,11 @@ class InProgressReverseSwaps {
 
   InProgressReverseSwaps(this._statuses, this.claimTxId);
 
-  int get lockupTxETA => (_statuses.paymentsStatus.isNotEmpty)
+  int get lockupTxETA => (_statuses?.paymentsStatus?.isNotEmpty == true)
       ? _statuses.paymentsStatus[0].eta
       : -1;
 
-  String get lockTxID => (_statuses.paymentsStatus.isNotEmpty)
+  String get lockTxID => (_statuses?.paymentsStatus?.isNotEmpty == true)
```
Comment (erdemyerebasmaz) [inline lib/bloc/reverse_swap/reverse_swap_model.dart:73]: Why is == true necessary?

Comment (erdemyerebasmaz) [reply lib/bloc/reverse_swap/reverse_swap_model.dart:73]: Okay, I've read your comment on other PR. Go ahead and merge.

**Review by ubbabeck — COMMENTED** [review]
Looks good to me. I will test with and on chain tx asap.

**Comment by kingonly** [comment]
@ubbabeck please confirm and merge when you can please

**Pull request merged**